src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -496,6 +516,31 @@ public void Resolve(SimulationParameters parameters)
         }
     }
 
+    /// <summary>
+    ///   A bbcode string containing all the unlock conditions for this organelle.
+    /// </summary>
+    public LocalizedStringBuilder GenerateUnlockRequirementsText(GameProperties game)
+    {
+        LocalizedStringBuilder unlockRequirements = new();
+        if (UnlockConditions != null)
+        {
+            bool first = true;
+            foreach (var unlockCondition in UnlockConditions)
+            {
+                if (!first)
+                {
+                    unlockRequirements.Append(new LocalizedString("UNLOCK_OR"));
```

Comment:
```suggestion
                    unlockRequirements.Append(new LocalizedString("UNLOCK_WITH_ANY_OF_FOLLOWING"));
```

I think having a really clear translation key here would be beneficial.

Reply:
`UNLOCK_WITH_ANY_OF_FOLLOWING` does not match what the translation is. The translation string is literally just `or`.

However this is a good suggestion for the current `UNLOCK_WITH` key.

Reply:
"or" is very likely much too context dependent. So the translation key needs to do the heavy lifting here. I do agree though that my suggestion doesn't really match this context then (I didn't read the translation text yet so I didn't know it was "or").

Reply:
Guess I'll rename it to something like `OR_UNLOCK_CONDITION` then